### ISSUE TITLE
library does not show modules and readme

### ISSUE THREAD

**Issue by m-bock** [dmy/elm-0.18-packages#1]
https://package.elm-lang.org/packages/thought2/elm-wikimedia-commons/1.1.0

somehow this library does not show up the readme nor the exposed modules, did I miss something?

https://github.com/thought2/elm-wikimedia-commons/tree/1.1.0

**Comment by rlefevre** [comment]
This is a bug in the official [package.elm-lang.org](https://github.com/elm/package.elm-lang.org) web site for (some?) 0.18 packages published after 0.19 release that do not have an equivalent version for 0.19. See https://github.com/elm/package.elm-lang.org/issues/281, however the problem does not appear anymore for the package in this bug as the same version has been published for 0.19. 

Feel free to add `thought2/elm-wikimedia/commons` to this bug as a new example.

Note that the package still installs correctly:
```
$ elm-package install thought2/elm-wikimedia-commons
To install thought2/elm-wikimedia-commons I would like to add the following
dependency to elm-package.json:

    "thought2/elm-wikimedia-commons": "1.1.0 <= v < 2.0.0"

May I add that to elm-package.json for you? [Y/n]

Some new packages are needed. Here is the upgrade plan.

  Install:
    elm-community/json-extra 2.7.0
    elm-lang/core 5.1.1
    elm-lang/html 2.0.0
    elm-lang/http 1.0.0
    elm-lang/virtual-dom 2.0.4
    thought2/elm-wikimedia-commons 1.1.0

Do you approve of this plan? [Y/n]
Starting downloads...

  ● elm-lang/http 1.0.0
  ● elm-lang/html 2.0.0
  ● elm-community/json-extra 2.7.0
  ● elm-lang/core 5.1.1
  ● elm-lang/virtual-dom 2.0.4
  ● thought2/elm-wikimedia-commons 1.1.0

Packages configured successfully!
```

**Comment by rlefevre** [comment]
I will close this issue as https://dmy.github.io/elm-0.18-packages correctly displays and links to the last 0.18 version of this package, and cannot work-around the issue of https://package.elm-lang.org without rendering itself the documentation, which is not its purpose.

**Comment by rlefevre** [comment]
I actually found a way to restore access to the working package.elm-lang.org website for elm < 0.19, so documentations are now all there, including those for packages published after 0.19 release:

https://old.elm.dmy.fr

For example your documentation:
https://old.elm.dmy.fr/packages/thought2/elm-wikimedia-commons/latest
